Lib/test/test_typing.py

```diff
@@ -4967,12 +4978,25 @@ class C(Generic[T]): pass
         self.assertEqual(get_args(collections.abc.Callable[[int], str]),
                          get_args(Callable[[int], str]))
         P = ParamSpec('P')
+        self.assertEqual(get_args(P), ())
+        self.assertEqual(get_args(P.args), ())
+        self.assertEqual(get_args(P.kwargs), ())
         self.assertEqual(get_args(Callable[P, int]), (P, int))
+        self.assertEqual(get_args(collections.abc.Callable[P, int]), (P, int))
```

Comment:
I'm honestly surprised this isn't tested. Is `get_args` with `collections.abc.Callable` tested elsewhere? I recall having added code specifically for it in `get_args`.

Reply:
Yes, there are other tests for `get_args(collections.abc.Callable[])`, but not with `ParamSpec`.

Link: https://github.com/python/cpython/blob/8605932471912139e4942330dcb4b12fb00e8fd3/Lib/test/test_typing.py#L4975-L4979